### ISSUE TITLE
Object Level Security: OLS Power BI

### ISSUE THREAD

**Issue by tschwenger** [TabularEditor/TabularEditor#1045]
!!! For Tabular Editor 3 questions, go here --> https://github.com/TabularEditor/TabularEditor3/issues !!!

**What are you trying to achieve?**
Not sure if this is a bug or a question:
Trying to add OLS via Tabular Editor through Source controlled PBI Shared Dataset

I have a Power BI Dataset that I'm managing through an Azure DevOps repo that has CI CD. Earlier in dev, I added a reader role with no restrictions. 

This week, I added another Read Role with Object Level Securty OLS, I noticed in the latest deployment, when I added OLS to the model via tabular editor, that the roles were not deployed even though the code was deployed. The json files are in the repo. 

I followed up and checked management studio and the model contained no roles when connecting. 

So I made a test shared dataset where I created roles in power bi, managed them in tabular editor and then published to the service. I refreshed management studio and saw the roles in for the test dataset. 

When comparing the folder structures, I noticed that the model I've been developing is missing a code snippet in the json file for the role. It's missing an annotation snippet like below:

  "annotations": [
    {
      "name": "PBI_Id",
      "value": "fb32ac89f1f64f01bab232431651443b"
    }
  ]

Long story short, it doesn't look like I'm able to add OLS from a working Power BI Shared dataset that I'm managing through azure dev ops. Do I need to xmla script out a create/replace statement for the role and run it?

Is there anyway to add the security groups to a shared dataset after "save to folder." Every thing else I've added to the model has worked, do I need to add an annotation?

Note: 
I'm trying to add the role to shared dataset managed through CI/CD modified from your process:
https://www.youtube.com/watch?v=BbYqcSUXOgU 

I added roles to the model well before they were defined by the business. I set up a test model in power BI and set up a role and noticed the role had an annotation in the json file whereas there was no such annotation in the power bi file.

**Comment by otykier** [comment]
Assuming you're using the Tabular Editor CLI to perform the deployment, please make sure that you include the `-R` switch (to indicate you want to deploy the roles that are specified in the code). More info [here](https://docs.tabulareditor.com/te2/Command-line-Options.html).
